indexing-hadoop/src/main/java/org/apache/druid/indexer/JobHelper.java

```
@@ -349,6 +352,22 @@ public static void ensurePaths(HadoopDruidIndexerConfig config)
 
   public static boolean runSingleJob(Jobby job, HadoopDruidIndexerConfig config)
   {
+    String hadoopJobId = job.submitAndGetHadoopJobId();
+    ObjectMapper objectMapper = new ObjectMapper();
+    String hadoopJobIdFileName = config.getHadoopJobIdFileName();
+
+    if (hadoopJobIdFileName != null) {
+      try {
+        objectMapper.writeValue(new OutputStreamWriter(
+            new FileOutputStream(new File(hadoopJobIdFileName)), StandardCharsets.UTF_8),
+            hadoopJobId);
+        log.info("MR job id: %s is written to the file: %s", hadoopJobId, hadoopJobIdFileName);
+      }
+      catch (IOException e) {
+        log.error(e, "Error writing job id: %s to the file: %s.", hadoopJobId, hadoopJobIdFileName);
```

I think this should be `log.warn` if we ignore this error.